cmd/agent/command/command.go

```diff
@@ -46,7 +46,7 @@ type SubcommandFactory func(globalParams *GlobalParams) []*cobra.Command
 // without secrets and logger disabled).
 func GetDefaultCoreBundleParams(globalParams *GlobalParams) core.BundleParams {
 	return core.BundleParams{
-		ConfigParams: config.NewAgentParamsWithoutSecrets(globalParams.ConfFilePath),
+		ConfigParams: config.NewAgentParams(globalParams.ConfFilePath),
 		LogParams:    log.ForOneShot(LoggerName, "off", true)}
 }
```

Comment:
Do you think it would make sense to alway set `SecretParams` to force the user who copy past the code to think about secret?

Reply:
I think if it's disabled by default we can omit it for simpler usage.